Tests/TensorFlowTests/ContextTests.swift

```diff
@@ -49,9 +49,41 @@ final class ContextTests: XCTestCase {
       }
     }
   }
+  
+  func testGaussianNoise() {
+    Context.local.learningPhase = .inference
+    let gaussianNoise = GaussianNoise<Float>(standardDeviation: 1.0)
+    let x = Tensor<Float>(repeating: 1.0, shape: [5, 5])
+    XCTAssertEqual(gaussianNoise(x), x)
+    withLearningPhase(LearningPhase.inference) {
+      XCTAssertEqual(gaussianNoise(x), x)
+      withLearningPhase(LearningPhase.training) {
+        XCTAssertNotEqual(gaussianNoise(x), x)
+      }
+      XCTAssertEqual(gaussianNoise(x), x)
+    }
+    XCTAssertEqual(gaussianNoise(x), x)
+  }
+
+  func testGaussianDropout() {
+    Context.local.learningPhase = .inference
+    let dropout = GaussianDropout<Float>(probability: 0.5)
+    let x = Tensor<Float>(repeating: 1.0, shape: [5, 5])
+    XCTAssertEqual(dropout(x), x)
+    withLearningPhase(LearningPhase.inference) {
+      XCTAssertEqual(dropout(x), x)
+      withLearningPhase(LearningPhase.training) {
+        XCTAssertNotEqual(dropout(x), x)
+      }
+      XCTAssertEqual(dropout(x), x)
+    }
+    XCTAssertEqual(dropout(x), x)
+  }
```

Comment:
I think `ContextTests.swift` is not good place to put `testGaussianNoise`/`testGaussianDropout`.
The purpose of this file is to test things about `Context`.
`Layer`'s tests should go into [LayerTests.swift](https://github.com/tensorflow/swift-apis/blob/master/Tests/TensorFlowTests/LayerTests.swift).

There originally is `testDropout`, but `Dropout` is not crucial in that test case.
It should be renamed to `testWithLearningPhase`.
(Also `testDropout` should go into LayerTests.swift)